### ISSUE TITLE
When generating a new app it starts off with accessibility issues because of lacking title

### ISSUE THREAD

**Issue by rkristelijn** [mui/toolpad#4760]
### Steps to reproduce

Steps:
1. Open this link to live example: (required)
2.
3.

### Current behavior

would like to add:

```ts
export const metadeta = {
  title: 'My Toolpad Core Next.js App',
  description: 'This is a sample app built with Toolpad Core and Next.js',
};
```

to `layout.tsx` in generator

### Expected behavior

no accessibility issues

### Context

![Image](https://github.com/user-attachments/assets/cd150760-b922-4ad1-a60e-4f521b526e35)

### Your environment

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @mui/envinfo` goes here.
```
</details>

**Search keywords**: accessibility generator
